pkg/clusterlogging/clusterlogforwarder_test.go

```diff
@@ -290,9 +290,10 @@ func TestClusterLogForwarderUpdate(t *testing.T) {
 		},
 		{
 			testClusterLogForwarder: buildInValidClusterLogForwarderBuilder(buildClusterLogForwarderClientWithDummyObject()),
-			expectedError:           "clusterlogforwarder cannot be deleted because it does not exist",
-			outputs:                 newOutputs,
-			pipelines:               newPipelines,
+			//nolint:lll
+			expectedError: `ClusterLogForwarder.logging.openshift.io "" is invalid: metadata.name: Required value: name is required`,
```

Comment:
This made me realize the builder's validate method doesn't check `errorMsg` at all, but I can make a separate PR for that